classroom/snippets/src/test/java/TestAddAliasToCourse.java

```diff
@@ -22,7 +22,7 @@
 public class TestAddAliasToCourse extends BaseTest {
 
   @Test
-  public void testAddCourseAlias() throws IOException {
+  public void testAddCourseAlias() throws Exception {
```

Comment:
Why the change to the generic `Exception`?

Reply:
The GoogleNetHttpTransport throws a GeneralSecurityException so the method could throw either of those. I changed it to Exception to reduce since IOException and GeneralSecurityException belong to the Exception class.

Reply:
If we know the specific exceptions that may occur (and the number is reasonable), IMO we should show them with explanation about why they might occur. This doesn't seem unreasonable:

`public void testAddCourseAlias() throws GeneralSecurityException, IOException {`

Reply:
Updated the method signature to include specific errors.